ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -108,9 +108,7 @@ tests:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
-        TEST=./discovery-infra/tests/test_kube_api.py
-        TEST_FUNC=test_kube_api_ipv4
-        POST_INSTALL_COMMAND="export KUBECONFIG=\$(find \${KUBECONFIG} -type f); make deploy_assisted_operator test_parallel"
+        POST_INSTALL_COMMAND="export TEST=./discovery-infra/tests/test_kube_api.py; export TEST_FUNC=test_kube_api_ipv4; export KUBECONFIG=\$(find \${KUBECONFIG} -type f); make deploy_assisted_operator test_parallel"
```

Comment:
this is pretty much unreadable and unstructured...
can we make it better?

Reply:
Done